### ISSUE TITLE
Will not download on a regular Pebble.

### ISSUE THREAD

**Issue by cyturralde** [tetrapus/PoGoPebble#6]
Neither the Watchface or the App will download on the Pebble. I get an alert that says it is only available for Steele or Time. Any chance of that changing?

**Comment by tetrapus** [comment]
Are you downloading the .pbw or using the app store?

**Comment by cyturralde** [comment]
Ok, it will install through the pbw but not through the app store. However, the background is just a dot, and it does not actually alert of any pokemon.

**Comment by nickidemic** [comment]
I'm experiencing the same thing. Also, flicking the wrist does nothing. When it finds a Pokemon, it displays some of the numbers briefly, then crashes.

**Comment by tetrapus** [comment]
From my experiments with the emulator I can't even get past the splash screen. It seems to be running out of memory, which is odd considering it's supposed to have the same amount as a PT. Could either of you be able to get app logs for me?

**Comment by tetrapus** [comment]
It may be necessary to create an OG branch that has b&w sprites so it doesn't blow the memory limits. I'm honestly not too keen on doing this with all the other features requests coming in, would anyone else be willing to give it a try?

**Comment by cyturralde** [comment]
If it's jist a matter of making the pics b&w I can do that in Photoshop I don't mind. As far as logs of you tell me how, I will.

**Comment by tetrapus** [comment]
Yeah don't worry, I'll do it :( I'll try to run a test tonight with a bunch
of black blobs to see if it makes it work, I'll update this thread if I
need any help with the real sprites but I have a script I can use to
automate most of the work

**Comment by tetrapus** [comment]
My script didn't work as well as I expected, but yeah, it looks like converting them to black and white works fine. If someone else could do this I'm happy to release it, the only constraints are that the images have to be as small as possible (<96KiB total) and black and white, preferrably dithered.

Thanks!

**Comment by nickidemic** [comment]
I'm willing to look at this, however I've never done dithering before. do you have a script that dithers one image file that you could send me?

**Comment by nickidemic** [comment]
k I'm totally done now. I have the images folder, ready to pass off back to you

**Comment by tetrapus** [comment]
Unfortunately those files are too big for the pebble classic. Would you be able to go through them all and shave off some bytes by switching from RGB to indexed mode? Make sure the palette is optimised so that it only contains black, white and transparent. I'm not sure what the process is for photoshop but in GIMP it's Image > Mode > Indexed

**Comment by nickidemic** [comment]
the best I can do is 114.4kB total
